camelot/admin/object_admin.py

```diff
@@ -671,7 +690,16 @@ class attribute.
                     choices_dict = dict(forced_attributes['choices'])
                     attributes['to_string'] = lambda x : choices_dict.get(x, '')
             self._expand_field_attributes(attributes, field_name)
-            return attributes
+
+        # Check for 'editable' & 'editable_fields' in __facade_args__, default is True
+        if isinstance(self.entity, EntityMeta):
```

Comment:
mogelijk zit deze check in dan in de 'get_descriptor_field_attributes' van de Entity admin op zijn plaats ?